src/libs/common/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+project(common)
```

Comment:
I'd suggest renaming 'common' into 'util'
Their semantics are similar, though we already have common_h (which I want to get rid of though)
Also, 'util' reflects its contents slightly better

Reply:
It was called _utils_ at first, then I changed it to _common_, so no problem changing it back to _util_.

Reply:
'utils' is maybe even better. it's up to you here :)